src/main/java/duke/Event.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
```

Comment:
Perhaps adding class level Javadoc comments would aid readers in understanding the function of each class better!